### PR TITLE
[engraving] Added LD_CONDITIONS to ... Step 65

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/engraving/rendering/dev/passlayoutindependentitems.cpp [hunk 1]
```diff
@@ -42,12 +42,16 @@ void PassLayoutIndependentItems::scan(EngravingItem* item, LayoutContext& ctx)
...
     case ElementType::AMBITUS:
     case ElementType::BREATH:
+    case ElementType::CLEF:
         TLayout::layoutItem(item, ctx);
     default:
         break;
     }
 
     for (EngravingItem* ch : item->childrenItems()) {
+        if (ch->isType(ElementType::DUMMY)) {
```
Comment (cbjeukendrup) [inline src/engraving/rendering/dev/passlayoutindependentitems.cpp:53]: Why is this change necessary? I'd think it would be weird if any other DummyElement was ever created other than "the" DummyElement, so how can we ever encounter it as a child somewhere?

**Review by cbjeukendrup — APPROVED** [review]

File: src/engraving/rendering/dev/passlayoutindependentitems.cpp (hunk 1 above)
Comment (igorkorsukov) [reply src/engraving/rendering/dev/passlayoutindependentitems.cpp:53]: The root element is RootItem, it has dummy in its children
